addons/survey/views/survey_templates_print.xml

```diff
@@ -28,7 +28,7 @@
                                 </t>
                                 <t t-if="not question.is_page and not answer or (question in answer.predefined_question_ids &amp; questions_to_display)" >
                                     <t t-set="answer_lines" t-value="answer.user_input_line_ids.filtered(lambda line: line.question_id == question)"/>
-                                    <div class="js_question-wrapper" t-att-id="question.id">
+                                    <div t-att-class="'js_question-wrapper col-md-12 col-sm-12 %s' % ('col-lg-12' if survey.questions_layout != 'page_per_question' else 'col-lg-6')" t-att-id="question.id">
```

Comment:
I would probably let this as it was before and keep the logic it the appropriate template. An issue I see is that we can have this situation https://tinyurl.com/25g2292u .

If we have 'one page per question', then we would have this https://tinyurl.com/25sfepcv, but I think this is too bad of an issue as it is just as reality but larger.